README.md

```diff
@@ -40,7 +40,17 @@ sudo snap install --edge nvim --classic
 
 [Other methods of installation are available in the official repository of Neovim](https://github.com/neovim/neovim/wiki/Installing-Neovim)
 
-Instructions for **NodeJS** are also available in [the official repo.](https://github.com/nodesource/distributions/blob/master/README.md)
+
+**NodeJS**
```

Comment:
Instead of
`**NodeJS**
There are several ways of installing NodeJS.
`
will be better to write
`
There are several ways of installing **NodeJS**:
`

Reply:
Done,